### PR TITLE
Update docker setup page - Closes #1605

### LINKED ISSUE #1605: Update Docker setup
Update the Docker setup for Lisk Core v4

- No big changes are expected, but check with lisk-core team to ensure all the steps are correct
- Config should be updated

#### Affected branch / URL
https://lisk.com/documentation/lisk-core/setup/docker.html

### PR DESCRIPTION
**Pull request by chris529**
Closes #1605

### CODE REVIEW

File: modules/ROOT/pages/setup/docker.adoc [hunk 1]
```diff
@@ -200,9 +204,9 @@ docker run -d --volume /data:/home/lisk/.lisk \
 
 [source,bash]
 ----
-docker run --volume lisk-data:/home/lisk/.lisk -it --rm lisk/core:3.0.4 blockchain:download --network=betanet --output=/home/lisk/.lisk/tmp/
-docker run --volume lisk-data:/home/lisk/.lisk -it --rm lisk/core:3.0.4 blockchain:import /home/lisk/.lisk/tmp/blockchain.db.tar.gz
-docker run --volume lisk-data:/home/lisk/.lisk -it --rm --entrypoint rm lisk/core:3.0.4 -f /home/lisk/.lisk/tmp/blockchain.db.tar.gz
+docker run --volume lisk-data:/home/lisk/.lisk -it --rm lisk/core:4.0.0 blockchain:download --network=betanet --output=/home/lisk/.lisk/tmp/
```
Comment (nagdahimanshu) [inline modules/ROOT/pages/setup/docker.adoc]:
```suggestion
docker run --volume lisk-data:/home/lisk/.lisk -it --rm lisk/core:4.0.0 blockchain download --network=betanet --output=/home/lisk/.lisk/tmp/
```

File: modules/ROOT/pages/setup/docker.adoc [hunk 2]
```diff
@@ -200,9 +204,9 @@ docker run -d --volume /data:/home/lisk/.lisk \
...
+docker run --volume lisk-data:/home/lisk/.lisk -it --rm lisk/core:4.0.0 blockchain:import /home/lisk/.lisk/tmp/blockchain.db.tar.gz
```
Comment (nagdahimanshu) [inline modules/ROOT/pages/setup/docker.adoc]:
```suggestion
docker run --volume lisk-data:/home/lisk/.lisk -it --rm lisk/core:4.0.0 blockchain import /home/lisk/.lisk/tmp/blockchain.db.tar.gz
```

File: modules/ROOT/pages/setup/docker.adoc [hunk 3]
```diff
@@ -13,10 +13,14 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 // Project URLs
+
+:url_config: management/configuration.adoc
+:url_upgrade_source: update/source.adoc
+:url_run_logging: lisk-core::run-blockchain/logging.adoc
 :url_run_logging: ROOT::run-blockchain/logging.adoc
 
 This document describes how to run Lisk Core as a Docker image-based container.
-Lisk Core version 3 does not have any external dependencies, and hence does not require using `docker-compose`.
+Lisk Core version 4 does not have any external dependencies, and hence does not require using `docker-compose`.
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/setup/docker.adoc]:
```suggestion
Lisk Core version 4 does not have any external dependencies and hence does not require using `docker-compose`.
```

File: modules/ROOT/pages/setup/docker.adoc [hunk 4]
```diff
@@ -13,10 +13,14 @@ Mona Bärenfänger <mona@lightcurve.io>
...
 
 == Pre-installation
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/setup/docker.adoc:26]: Line 52
`Please refer to the {url_docker_install_linux}[Docker installation for ubuntu]`

File: modules/ROOT/pages/setup/docker.adoc [hunk 5]
```diff
@@ -149,7 +153,7 @@ Testnet::
 docker run --volume lisk-data:/home/lisk/.lisk \
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/setup/docker.adoc:155]: Line 144
Please comment this line and add a TODO regarding updating hyperlinks to the page.

File: modules/ROOT/pages/setup/docker.adoc (hunk 4 above)
Comment (TalhaMaliktz) [inline modules/ROOT/pages/setup/docker.adoc:26]: It is unclear where the User should be added, if the user is being added to the OS then similar steps should be mentioned here as they are described in the [Source Setup Guide](https://liskhq.github.io/lisk-docs/lisk-core/setup/source.html#create-a-new-user).

File: modules/ROOT/pages/setup/docker.adoc [hunk 6]
```diff
@@ -79,7 +83,7 @@ Please refer to {url_docker_install_windows}[Docker installation for Windows^]
 ----
 git clone https://github.com/LiskHQ/lisk-core.git #<1>
 cd lisk-core/docker                  #<2>
-git checkout v3.0.4 -b v3.0.4 #<3>
+git checkout v4.0.0 -b v4.0.0 #<3>
 ----
 
 <1> Clone the repository.
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/setup/docker.adoc:90]: Line 90
<2> Navigate into the `lisk-core` root folder.

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
Looking good! Just a few suggestion :)

File: modules/ROOT/pages/setup/docker.adoc [hunk 7]
```diff
@@ -138,6 +143,7 @@ start --network mainnet --console-log=info
 ----
 
 See the xref:{url_run_logging}[] guide for more information about logging.
```
Comment (TalhaMaliktz) [inline modules/ROOT/pages/setup/docker.adoc]:
```suggestion
// See the xref:{url_run_logging}[] guide for more information about logging.
```

**Review by TalhaMaliktz — DISMISSED** [review]
Thanks, it LGTM 🥳. Just one small change though we need to comment out this line as the page it is referring to has not been updated yet.

**Review by nagdahimanshu — APPROVED** [review]
We are not able to test these changes with `lisk-core` v4.0.0 but changes looks good to me 👍. Hence, approving the PR.

**Pull request merged**